.github/workflows/test-report.yml

```
@@ -0,0 +1,272 @@
+              fs.writeFileSync('test-comment.md', fallbackComment);
+              
+              const output = process.env.GITHUB_OUTPUT;
+              if (output) {
+                fs.appendFileSync(output, 'test_status=error\\n');
+              }
+            }
+          "
+        
+      - name: Upload coverage to GitHub Pages
+        if: github.event_name == 'pull_request' && hashFiles('coverage/index.html') != ''
+        uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.7)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)